modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -3010,8 +3010,7 @@ private void _validateObjectStateTransition(
 		ListTypeEntry originalListTypeEntry =
 			_listTypeEntryLocalService.getListTypeEntry(
 				listTypeDefinitionId,
-				_getValue(
-					String.valueOf(values.get(entry.getKey()))));
+				_getValue(String.valueOf(values.get(entry.getKey()))));
```

Comment:
Did you really change this or was it by mistake?

Reply:
Not really, probably a change from SF.